Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -179,3 +203,22 @@ public struct PostgresBindings: Sendable, Hashable {
         self.metadata.append(.init(dataType: postgresData.type, format: .binary))
     }
 }
+
+extension PostgresBindings: CustomStringConvertible {
+    /// See ``Swift/CustomStringConvertible/description``.
+    public var description: String {
+        "[\(self.metadata.map { "\($0)" }.joined(separator: ", "))]"
+    }
+}
+
+extension PostgresBindings: CustomDebugStringConvertible {
+    /// See ``Swift/CustomDebugStringConvertible/debugDescription``.
+    public var debugDescription: String {
+        """
+        PostgresBindings(\
+        metadata: [\(self.metadata.map { String(reflecting: $0) }.joined(separator: "; "))], \
+        bytes: \(self.bytes.readableBytes) [\(Array(self.bytes.readableBytesView).prefix(8).map(\.description).joined(separator: ","))\(self.bytes.readableBytes > 8 ?",..." : "")]\
+        )
+        """
```

Comment:
I think this leaks an internal impl. detail that the user doesn't have to be aware of...

I like the array representation in `description`. Can we go for something like this:

```
[(type: uuid, format: bin, 16 bytes), (type: int4, format: bin, 4 bytes)]
```

Later we could present the actual values, if we wanted to. This could make debugging a lot easier!
```
[(type: uuid, format: bin, value: "45223d46-c8ca-4c90-9ad1-8a829db6a7ba"), (int4, bin, value: 12)]
```

Reply:
Both times we'd have to actually parse out the buffer just to generate the description. That's a lot of work.